OsuPlayer/Views/SettingsView.axaml.cs

```diff
@@ -64,7 +64,7 @@ public async void ImportSongsClick(object? sender, RoutedEventArgs routedEventAr
             return;
         }
 
-        var dbFilePath = result.First().Path.ToString();
+        var dbFilePath = result.First().Path.AbsolutePath;
```

Comment:
You changed this here, can you check the other 2 instances where we access Files and Folders (Import and Export) and change it to `AbsolutePath` there as well?

Then it is all the same everywhere